sshd-sftp/src/main/java/org/apache/sshd/sftp/server/SftpSubsystem.java

```diff
@@ -893,6 +893,9 @@ protected void doClose(int id, String handle) throws IOException {
             nodeHandle.close();
             listener.closed(session, handle, nodeHandle, null);
         } catch (IOException | RuntimeException | Error e) {
+            if (log.isTraceEnabled()) {
+                log.debug("doClose({})[id={}] Exception", session, id, e);
+            }
```

Comment:
This is a similar case as the doRename() over in AbstractSftpSubsystemHelper. Log-then-throw should not be necessary; IOException and RuntimeException will be logged in the sendStatus() call in doClose(Buffer, int), and for Error there should be last-resort handler somewhere higher up that logs it.

Besides, it should be log.isDebugEnabled(), if at all.

Reply:
done.